tests/roots/test-root/objects.txt

```diff
@@ -180,7 +180,9 @@ Others
 
 .. option:: arg
 
-Link to :option:`perl +p`, :option:`--ObjC++`, :option:`--plugin.option`, :option:`create-auth-token` and :option:`arg`
+.. option:: -j=[N]
+
+Link to :option:`perl +p`, :option:`--ObjC++`, :option:`--plugin.option`, :option:`create-auth-token`, :option:`arg` and :option:`-j`
```

Comment:
Adding an example does not test anything. Please add an expected result HTML tag to here.
https://github.com/sphinx-doc/sphinx/blob/cee921fe416068021996ec24ad19afb54cd6bbec/tests/test_build_html.py#L321-L333